c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,10 +80,13 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
-    }));
+    var mapSize = this.map.getSize();
+    if (mapSize) {
+      this.map.getView().fit(app.MapController.DEFAULT_EXTENT, mapSize,
+      /** @type {olx.view.FitOptions} */ ({
```

Comment:
> Should padding be a required attribute?

Maybe but the API doc also says the default value is ``[0, 0, 0, 0]`` so I guess it is not that required to specify it?